packages/api/pw-test.config.cjs

```diff
@@ -53,7 +53,7 @@ module.exports = {
     await execa(cli, ['db', '--start'])
     console.log('⚡️ Cluster and Postgres started.')
 
-    await execa(cli, ['db-sql', '--cargo', '--testing', '--reset'])
+    await execa(cli, ['db-sql', '--cargo', '--testing'])
```

Comment:
What's the reason for this change please?

Reply:
I am refactoring dev, this should've went over to that branch, I'll remove this change.

https://github.com/nftstorage/nft.storage/pull/1485